packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -260,6 +265,46 @@ test.describe('PieCookieBanner - Component tests', () => {
         expect(isToggleAllChecked).toBe(false);
     });
 
+    test.describe('`locale` prop', () => {
+        test('should render text in the default (English) language when the locale is not set', async ({ mount, page }) => {
+            // Arrange
+            await mount(PieCookieBanner, {});
+
+            // Act
+            const acceptAllButton = page.locator(acceptAllSelector);
+            const necessaryOnlyButton = page.locator(necessaryOnlySelector);
+            const managePreferencesButton = page.locator(managePreferencesSelector);
+            const componentDescription = page.locator(componentDescriptionSelector);
+            const modalDescription = page.locator(modalDescriptionSelector);
+
+            // Assert
+            expect(await acceptAllButton.textContent()).toContain('Accept all');
```

Comment:
These are fine for now, but I wonder if in future we can do something like this:
```suggestion
            expect(await acceptAllButton.textContent().trim()).toEqual(englishLocale.banner.cta.acceptAll);
```

In my opinion, the test shouldn't necessarily be concerned what the text is, just that it matches what's in the translation files. What do you think?